crawlers/ten_ago_dbcnt.py

```
@@ -0,0 +1,59 @@
+            else:
+                print("⚠️ 한국어/영어 외 기사입니다. 스킵합니다.")
+                continue
+
+            print(f"\n📰 제목: {headline}")
+            print(f"\n📝 요약: {summary}")
+
+            # 4. DB 저장
+            insert_summary_to_mysql(category, url, headline, summary, article_date)
+            print(f"✅ [{category}] 저장 완료")
+
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)